### ISSUE TITLE
[a11y] On startup, multiple views focused, causing redundant announcements

### ISSUE THREAD

**Issue by mcomella** [mozilla-mobile/firefox-echo-show#194]
STR:
- Open app with voiceView enabled

Expected: "Search or enter address" or "Search with Google"
Actual: "Search or enter address; Search with Google"

The double announcement is probably annoying but not a deal breaker. This only happens on startup: this calls into question what should be focused on startup (#195).

This is not a regression from the slide up overlay refactor (#117) though the text changed after #162 to "Search with Google" instead of "Search with Google or enter address".

Note: I struggled to fix this during #162: it was not as easy as I expected.

**Comment by athomasmoz** [comment]
Duplicate of #195
